### ISSUE TITLE
Can't find Task-adaptive Fine-tuning Inference(TFI) .

### ISSUE THREAD

**Issue by xinyanghuang7** [slei109/PATNet#5]
Hello, Author. 

When I reproduced the fine-tuning stage, I **did not find the TFI in the code** you provided, but I **found the finetune_reference you defined in patnet.py.**

So I tried to add it to test.py, but it didn't work. I think it may be because pred_mask can't request gradient.

I hope you can answer my questions or add the complete code, thank you!

**Comment by slei109** [comment]
Thanks for your interest. Yes, test.py is for testing without fine-tuning. It's for better understanding our problem settings and reimplementing the results for the main idea of the model. We are working on optimizing the TFI strategy for the following work. Once we finish, we will release the code.
